lms/djangoapps/instructor_task/api.py

```diff
@@ -521,3 +545,30 @@ def regenerate_certificates(request, course_key, statuses_to_regenerate):
     )
 
     return instructor_task
+
+
+def export_assignment_grades_csv(request, course_key, assignment_name):
+    """
+    Submits a task to generate a CSV grade report for an assignment.
+    """
+    task_type = 'export_assignment_grades_csv'
+    task_class = export_assignment_grades_csv_task
+    task_input = {
+        "assignment_name": assignment_name
+    }
+    task_key = ""
+    return submit_task(request, task_type, task_class, course_key, task_input, task_key)
+
+
+def export_grades_to_rgb(request, course_key, assignment_name, email):
+    """
+    Submits a task to export assignment grades to a remote gradebook.
+    """
+    task_type = TASK_TYPE_EXPORT_GRADES_TO_RGB
+    task_class = export_grades_to_rgb_task
+    task_input = {
+        "assignment_name": assignment_name,
+        "email_id": email
```

Comment:
Please change `email_id` to `email`

Reply:
it will create an issue, a confusion please see `extract_task_features`, it will mark task status incomplete 

- https://github.com/mitodl/edx-platform/blob/master/lms/djangoapps/instructor/views/api.py#L2331
-https://github.com/mitodl/edx-platform/blob/master/lms/djangoapps/instructor_task/views.py#L150

Reply:
OK, we can leave as it is